src/main/java/com/pinterest/secor/io/KeyValue.java

```diff
@@ -25,16 +25,29 @@
  */
 public class KeyValue {
 	
-	private final long mKey;
+	private final long mOffset;
+	private final byte[] mKey;
```

Comment:
so there would be getKey() which returns the offset, and getKafkaKey() which returns the key?  That seems more confusing.